Jint/Options.cs

```diff
@@ -120,22 +120,39 @@ internal void Apply(Engine engine)
                         (thisObj, arguments) =>
                             new NamespaceReference(engine, TypeConverter.ToString(arguments.At(0)))),
                     PropertyFlag.AllForbidden));
-                engine.Realm.GlobalObject.SetProperty("unwrapClr", new PropertyDescriptor(new ClrFunctionInstance(
-                    engine,
-                    "unwrapClr",
-                    (thisObj, arguments) =>
-                    {
-                        var arg = arguments.At(0);
-                        if (arg is ObjectWrapper obj)
-                        {
-                            return new ObjectWrapper(engine, obj.Target);
-                        }
-                        else
-                        {
-                            return arg;
-                        }
-                    }),
+                engine.Realm.GlobalObject.SetProperty("clrToString", new PropertyDescriptor(new ClrFunctionInstance(
```

Comment:
as the function name already changes, would it make sense to inject and object like `clrHelper` which would have these functions exposed? it could take `allowGetType` when constructed and throw error when `clrType` etc are being called. Usage would then be

`const unwrapped = clrHelper.unwrap(myObj);`

Reply:
Almost done except Tests.